network/topics/sub_filter.go

```diff
@@ -43,13 +44,25 @@ func (sf *subFilter) CanSubscribe(topic string) bool {
 // FilterIncomingSubscriptions is invoked for all RPCs containing subscription notifications.
 // It should filter only the subscriptions of interest and my return an error if (for instance)
 // there are too many subscriptions.
-func (sf *subFilter) FilterIncomingSubscriptions(pi peer.ID, subs []*ps_pb.RPC_SubOpts) ([]*ps_pb.RPC_SubOpts, error) {
+func (sf *subFilter) FilterIncomingSubscriptions(pi peer.ID, subs []*ps_pb.RPC_SubOpts) (res []*ps_pb.RPC_SubOpts, err error) {
+	defer func() {
+		zap.L().Debug(
+			"FilterIncomingSubscriptions",
+			fields.PeerID(pi),
+			zap.Int("before_subscriptions_len", len(subs)),
+			zap.Int("after_subscriptions_len", len(res)),
+			zap.Any("before_subscriptions", subs),
```

Comment:
This log is super verbose, we should probably not print all the subs array. either make it smaller or remove.